fastchat/serve/api_provider.py

```diff
@@ -129,9 +140,21 @@ def openai_api_stream_iter(
         model_name = "gpt-4-1106-preview"
```

Comment:
A few lines above, should the api_version for "azure" be updated?

Reply:
I'm less familiar with the OpenAI on Azure, but looks like from [these docs](https://learn.microsoft.com/en-us/azure/ai-services/openai/how-to/gpt-with-vision?source=recommendations&tabs=python%2Csystem-assigned%2Cresource) that the newest version for GPT-4-Turbo Vision is `2023-12-01-preview`? However, there are multiple versions depending on which model (e.g. gpt3.5 vs. gpt4) so could be a bit more complicated. Do you happen to know if we still use the Azure endpoints and if so what version we should use @infwinston?